spec/support/bin/@registry_test

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
```

Comment:
Why the `@` in the filename?

Reply:
Again, worth adding a comment at the top of this script just to document the positional arguments and usage. e.g. 
```
# Runs the `snyk` command authenticated with Snyk and with the OCI Registry URLs set.
# It expects the `SNYK_TOKEN` environment variable to be set to ... and the `SHELLSPEC_SUPPORT_BIN`
# environment variable to be set to ...
#
# Usage:
#     % ./@registry_test <registry_url> <registry_username> <registry_password>
```